mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/control/plane/resource/cluster/proto/ListResourceClusterRequest.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.mantisrx.control.plane.resource.cluster.proto;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Builder
+@Value
+public class ListResourceClusterRequest {
```

Comment:
move these within the actor class.

Reply:
All these contracts in a single file are going to make it a huge one though? Also, there are composite referenced classes that are already huge by themselves.